### PR TITLE
[MIG][14.0] account_bank_statement_import_oca_camt54

### PR DESCRIPTION
**Pull request by i-vyshnevska**
depends 
 - [X] https://github.com/OCA/bank-statement-import/pull/362
- [X] https://github.com/OCA/bank-statement-import/pull/367

### CODE REVIEW

**Comment by damdam-s** [comment]
now depends on #369 and #367 both merged

File: account_statement_import_camt54/README.rst [hunk 1]
```diff
@@ -0,0 +1,90 @@
+==========================
```
Comment (damdam-s) [inline account_statement_import_camt54/README.rst:1]: this file needs to be regenerated to have the good module name and Odoo main version

File: account_statement_import_camt54/models/account_journal.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+from odoo import fields, models
+
+
+class AccountBankStatementImport(models.Model):
+
+    _inherit = "account.journal"
+
+    transfer_line = fields.Boolean(
+        string="Add balance Line",
+        help="Generate balance line on total of bank statemen import",
```
Comment (damdam-s) [inline account_statement_import_camt54/models/account_journal.py]:
```suggestion
        help="Generate balance line on total of bank statement import",
```

File: account_statement_import_camt54/models/account_statement_import.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    def _complete_stmts_vals(self, stmts_vals, journal, account_number):
+        """Search partner from partner reference"""
+        stmts_vals = super()._complete_stmts_vals(stmts_vals, journal, account_number)
+        for st_vals in stmts_vals:
+            for line_vals in st_vals["transactions"]:
+                if "partner_ref" in line_vals:
+                    partner_ref = line_vals.pop("partner_ref")
+                    partner = self.env["res.partner"].search(
+                        [("ref", "=", partner_ref)], limit=1
+                    )
```
Comment (damdam-s) [inline account_statement_import_camt54/models/account_statement_import.py:48]: log something or raise an error if not partner found?

**Review by damdam-s — APPROVED** [review]
pre approved

File: account_statement_import_camt54/README.rst (hunk 1 above)
Comment (i-vyshnevska) [reply account_statement_import_camt54/README.rst:1]: regeneration will be done by bot, you need to take care only on the files in readme folder

File: account_statement_import_camt54/models/account_statement_import.py (hunk 3 above)
Comment (i-vyshnevska) [reply account_statement_import_camt54/models/account_statement_import.py:48]: it is an order case when partner is not set

**Review by ajaniszewska-dev — APPROVED** [review]

**Comment by sonhd91** [comment]
Hello @i-vyshnevska, we noticed that `transfer_line` field was added but never shown on form, should we add it or did we miss something ?

**Comment by i-vyshnevska** [comment]
@sonhd91 indeed UI changes are missing, they need to be added to journal form

**Pull request merged**